scout/adapter/mongo/variant.py

```diff
@@ -689,39 +689,43 @@ def delete_variants(self, case_id, variant_type, category=None):
         result = self.variant_collection.delete_many(query)
         LOG.info("{0} variants deleted".format(result.deleted_count))
 
-    def overlapping(self, variant_obj, limit=30):
+    def hgnc_overlapping(self, variant_obj: dict, limit: int = None) -> Iterable[Dict]:
```

Comment:
I get sooo confused each time I read overlapping, also specifically about "overlapping SVs" thinking only about the SV inexact matching. The users are sometimes as well, so a slight qualification both in code and html is called for.